spec/examples/record/callbacks_spec.rb

```diff
@@ -35,12 +35,12 @@ def executed_callbacks
 
     attr_writer :instance_after_save
 
-    private
-
     def create_post
       post = Post.create!(permalink: 'autopost', title: 'Auto Post')
       self.post = post
     end
+    
+    private    
```

Comment:
Does `#create_post` need to be public?

Reply:
I don't remember moving this code (maybe I did).. so I moved it back.